src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2416,8 +2415,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
                     // Compute the location of the top left corner of the cell in DIPS
                     const til::size marginsInDips{ til::math::rounding, GetPadding().Left, GetPadding().Top };
-                    const til::point startPos{ lastHoveredCell->X,
-                                               lastHoveredCell->Y };
+                    const til::point startPos{ lastHoveredCell->x(),

```

Comment:
is this the same as `const auto startPos = *lastHoveredCell;` ? It looks like we're making a til::point from a til::point inside an optional. :smile: